shell/client/shell.html

```diff
@@ -186,6 +190,30 @@
   {{/with}}
 </template>
 
+<template name="requestAccess">
+  <div class="grain-interstitial">
+  <p>You currently do not have permission to access this grain.</p>
+  {{#if currentUser}}
+    {{#with status}}
+      {{#if showButton}}
+        <button class="request-access">Request Access</button>
+      {{/if}}
+      {{#if waiting}}
+         <p>Sending your request...</p>
+      {{/if}}
+      {{#if success}}
+         <p>Your request has been sent.</p>
```

Comment:
Consider setting expectations about how the user will know if/when they have been granted access to the grain - will they receive an email?  Will it appear silently in their grain list?